src/test/kotlin/com/personio/synthetics/builder/AssertionsBuilderTest.kt

```diff
@@ -67,6 +67,27 @@ class AssertionsBuilderTest {
         )
     }
 
+    @Test
+    fun `bodyContainsJsonPathRegex adds assertion that macthes the value stored in the given json path against the regex provided`() {
+        assertionsBuilder.bodyContainsJsonPathRegex("any_json_path", "any_regex")
+        val result = assertionsBuilder.build()
+
+        assertEquals(
+            SyntheticsAssertion(
+                SyntheticsAssertionJSONPathTarget()
+                    .operator(SyntheticsAssertionJSONPathOperator.VALIDATES_JSON_PATH)
+                    .type(SyntheticsAssertionType.BODY)
+                    .target(
+                        SyntheticsAssertionJSONPathTargetTarget()
+                            .jsonPath("any_json_path")
```

Comment:
`any_json_path` is not a good example of a JSON path.

e.g.

Using the below JSON string:

```
{
  "foo": {
    "bar": "baz"
  }
}
```

An example path would be `foo.bar`


Because `any_json_path` uses underscores, it doesn't represent atypical JSON path, in my opinion.

Reply:
Makes sense to have more relatable test data. Updated it ✅